src/main/java/org/mule/extension/file/api/LocalFileAttributes.java

```diff
@@ -26,12 +27,25 @@
  */
 public class LocalFileAttributes extends AbstractFileAttributes {
 
+  @Parameter
```

Comment:
not sure if this can be null or not TBH

Reply:
https://docs.oracle.com/javase/7/docs/api/java/nio/file/attribute/BasicFileAttributes.html#lastModifiedTime()